Android/AmazonConnectInAppCallingAndroidSample/app/src/main/java/com/amazonaws/services/connect/inappcalling/sample/service/MicrophoneService.kt

```diff
@@ -0,0 +1,70 @@
+package com.amazonaws.services.connect.inappcalling.sample.service
```

Comment:
Is this something that we should also add to Chime SDK? or at least as a demo?

Reply:
We already did